packages/eslint-config-udemy-react-addons/parts/react-hooks.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = {
+    plugins: ['react-hooks'],
+    rules: {
+        // Only call Hooks from react functions and only call Hooks at the top level
+        'react-hooks/rules-of-hooks': 'error',
+        // Verifies the list of dependencies for Hooks like useEffect and similar
+        'react-hooks/exhaustive-deps': 'warn',
```

Comment:
Might as well make this `error`. Looks like messing up the dependencies will break your code in subtle ways, so better to make people deal with the issue rather than having it be a warning that can be ignored.

Reply:
Okay, I made it an error.